### PR TITLE
Add agama interactive dummy test

### PR DESCRIPTION
**Pull request by lemon-suse**
- Related ticket: https://progress.opensuse.org/issues/163943
- Needles:  N/A
- Verification run: https://openqa.opensuse.org/tests/4415036#step/agama_is_visible/1

### CODE REVIEW

File: schedule/yam/agama_interactive.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (jknphy) [inline schedule/yam/agama_interactive.yaml:1]: why not just agama? is taken? can we replace it?
s/nteractive/interactive/
In theory interactive is the default, so no need to mention.

File: schedule/yam/agama_interactive.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+name: agama_interactive
+description: >
+  Perform nteractive installation with agama.
+schedule:
+  - installation/bootloader_start
+  - yam/agama/dummy
```
Comment (jknphy) [inline schedule/yam/agama_interactive.yaml:7]: let's call it `agama_is_visible.pm` but whatever you like more... :)

File: tests/yam/agama/dummy.pm [hunk 3]
```diff
@@ -0,0 +1,17 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: First installation using D-Installer current CLI (only for development purpose)
```
Comment (jknphy) [inline tests/yam/agama/dummy.pm:4]: please update this.

File: schedule/yam/agama_interactive.yaml (hunk 1 above)
Comment (lemon-suse) [reply schedule/yam/agama_interactive.yaml]: Updated it to agama.yaml.

File: schedule/yam/agama_interactive.yaml (hunk 2 above)
Comment (lemon-suse) [reply schedule/yam/agama_interactive.yaml]: Ok, NP for me.

File: tests/yam/agama/dummy.pm (hunk 3 above)
Comment (lemon-suse) [reply tests/yam/agama/dummy.pm]: Updated.

**Pull request merged**